src/test/java/org/broadinstitute/hellbender/tools/spark/sv/evidence/XGBoostEvidenceFilterUnitTest.java

```diff
@@ -0,0 +1,427 @@
+        };
+
+        SortedMap<String,Integer> contigLengthMap = new TreeMap<>(contigComparator);
+        final String svGenomeGapsFile = params.svGenomeGapsFile;
+        int lineNumber = -1;
+        try(final InputStream inputStream = XGBoostEvidenceFilter.getInputStream(svGenomeGapsFile)) {
+            final BufferedReader reader = new BufferedReader(new InputStreamReader(inputStream));
+            String line;
+            while((line = reader.readLine()) != null) {
+                ++lineNumber;
+                if(line.startsWith("#")) {
+                    continue;
+                }
+                final String[] splitLine = line.split("\t");
+                final String contig = splitLine[0];
+                final Integer end = Integer.parseInt(splitLine[2]) + 1;
+                if(!contigLengthMap.containsKey(contig) || contigLengthMap.get(contig) < end) {
+                    contigLengthMap.put(contig, end);
+                }
+            }
+        } catch(IOException e) {
+            throw new GATKException("Unable to read genome gaps file \"" + svGenomeGapsFile + "\" line " + lineNumber
+                                    + ": " + e.getMessage());
+        }
+
+        SAMFileHeader header = new SAMFileHeader();
+        header.setSortOrder(SAMFileHeader.SortOrder.coordinate);
+        SAMSequenceDictionary dict = new SAMSequenceDictionary();
+        // make up some sequence records
+        for (SortedMap.Entry<String,Integer> contigEntry : contigLengthMap.entrySet()) {
+            final String contig = contigEntry.getKey();
+            final int chromosomeSize = contigEntry.getValue();
+            SAMSequenceRecord rec = new SAMSequenceRecord(contig, chromosomeSize);
+            rec.setSequenceLength(chromosomeSize);
+            dict.addSequence(rec);
+        }
+        header.setSequenceDictionary(dict);
+        return header;
+    }
+
+    private static ReadMetadata initMetadata() {
+        final ReadMetadata.PartitionBounds[] partitionBounds = new ReadMetadata.PartitionBounds[3];
+        partitionBounds[0] = new ReadMetadata.PartitionBounds(0, 1, 0, 10000, 9999);
+        partitionBounds[1] = new ReadMetadata.PartitionBounds(0, 10001, 0, 20000, 9999);
+        partitionBounds[2] = new ReadMetadata.PartitionBounds(0, 20001, 0, 30000, 9999);
+        return new ReadMetadata(Collections.emptySet(), artificialSamHeader,
+                new LibraryStatistics(cumulativeCountsToCDF(featuresTestData.template_size_cumulative_counts),
+                        60000000000L, 600000000L, 1200000000000L, 3000000000L),
+                partitionBounds, 100, 10, featuresTestData.coverage);
+    }
+
+    private static IntHistogram.CDF cumulativeCountsToCDF(final long[] cumulativeCounts) {
+        final float[] cdfFractions = new float[cumulativeCounts.length];
+        final long totalObservations = cumulativeCounts[cumulativeCounts.length - 1];
+        for(int index = 0; index < cdfFractions.length; ++index) {
```

Comment:
`cdfFractions = Arrays.stream(cumulativeCounts).map(n -> n / totalObservations).toArray()`

Reply:
Unfortunately can't stream this because cdfFraction must be *float* and there's no mapToFloat function.